app/viewmodels/DutyDefermentStatementsForEori.scala

```diff
@@ -20,19 +20,27 @@ import models.{DutyDefermentStatementFile, EoriHistory}
 import utils.DateConverters.OrderedLocalDate
 import utils.OrderedByEoriHistory
 
+import java.time.LocalDate
+
 case class DutyDefermentStatementsForEori(eoriHistory: EoriHistory,
                                           currentStatements: Seq[DutyDefermentStatementFile],
                                           requestedStatements: Seq[DutyDefermentStatementFile])
   extends OrderedByEoriHistory[DutyDefermentStatementsForEori] {
 
+  private val numberOfMonths: Int = 7
+  private val endDate = LocalDate.now()
+  private val startDate: LocalDate = endDate.minusMonths(numberOfMonths).withDayOfMonth(1)
```

Comment:
Better to move this into a common method (inside Utils) and keep method name like firstDayOfPastNthMonth or firstDayNthMonthAgo or on the lines that conveys the best meaning and write tests